### PR TITLE
fix(@schematics/angular): update `ng-packagr` and `tsickle` when migr…

### PR DESCRIPTION
**Pull request by alan-agius4**
…ating to version 8

### CODE REVIEW

File: packages/schematics/angular/migrations/update-8/update-dependencies.ts [hunk 1]
```diff
@@ -11,55 +11,29 @@ import { latestVersions } from '../../utility/latest-versions';
...
-          type: current.type,
-          name: '@angular-devkit/build-ng-packagr',
-          version: latestVersions.DevkitBuildNgPackagr,
-          overwrite: true,
-        },
-      );
-    }
+    const dependenciesToUpdate: Record<string, string> = {
+      '@angular-devkit/build-angular': latestVersions.DevkitBuildAngular,
+      '@angular-devkit/build-ng-packagr': latestVersions.DevkitBuildNgPackagr,
```
Comment (clydin) [inline packages/schematics/angular/migrations/update-8/update-dependencies.ts]: can you also add `'@angular-devkit/build-webpack'` as well?

Comment (alan-agius4) [reply packages/schematics/angular/migrations/update-8/update-dependencies.ts]: Added

File: packages/schematics/angular/utility/latest-versions.ts [hunk 2]
```diff
@@ -16,4 +16,8 @@ export const latestVersions = {
   // The versions below must be manually updated when making a new devkit release.
   DevkitBuildAngular: '~0.800.0-beta.13',
   DevkitBuildNgPackagr: '~0.800.0-beta.13',
+  DevkitBuildWebpack: '~0.800.0-beta.13',
```
Comment (clydin) [inline packages/schematics/angular/utility/latest-versions.ts]: The beta specifier shouldn’t be needed anymore.

Comment (alan-agius4) [reply packages/schematics/angular/utility/latest-versions.ts]: Oops, PTAL

**Pull request merged**